buddyboss/buddyboss-platform
# PROD-712#3088 Fix Video upload progress bar showing 100% and still it does not allow post update due to confusion

Its the confusion with the actual file data been send to server shows "100% uploaded" but after that it takes time for our background process to save the attachment and create DB record etc. and then throws the response in the success event, so another approach is to move the lines of code for showing 100% uploaded from the "uploadprogress" event to the actual completion "success" event handler

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3088 .

### How to test the changes in this Pull Request:

1. Please check on server not on localhost
2. Goto activity feed and upload the large video, so it takes time to upload
2. Earlier as soon as the progress bar shows 100%, if clicked post update button it gave error, now check with this PR it will be no such issue
3. See error

### Proof Screenshots or Video
http://somup.com/cr6oX433Sq at time 6:20

After fix
https://somup.com/crX0Qa0FrU

### Other information:




### Changelog entry

> Moved "100% uploaded" display condition on the success event instead of uploadprogress event handler
